<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EconomistDownloader</class>
 <widget class="QMainWindow" name="EconomistDownloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>544</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>982</width>
    <height>544</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>The Economist Downloader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,2">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="sizeIncrement">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>80</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../image.qrc">:/newPrefix/img/Logo.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_4">
        <property name="minimumSize">
         <size>
          <width>398</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="Developed">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>0</y>
           <width>311</width>
           <height>51</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Arial Black</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#55557f;&quot;&gt;Developed by Yiang Fan&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Economist</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="text">
                   <string>Year</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="text">
                   <string>Month</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="text">
                   <string>Day</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="text">
                   <string>Issue</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,1">
                <item>
                 <widget class="QLineEdit" name="lineEdit"/>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox">
                  <property name="font">
                   <font>
                    <family>Arial</family>
                    <pointsize>10</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">
QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
}
</string>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                  <property name="currentText">
                   <string>--Please select Month--</string>
                  </property>
                  <property name="duplicatesEnabled">
                   <bool>true</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>--Please select Month--</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>01</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>02</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>03</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>04</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>05</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>06</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>07</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>08</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>09</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>11</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>12</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_2">
                  <property name="styleSheet">
                   <string notr="true">QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
}
</string>
                  </property>
                  <property name="editable">
                   <bool>true</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>--Please select Day--</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>01</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>02</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>03</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>04</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>05</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>06</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>07</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>08</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>09</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>10</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>11</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>12</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>13</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>14</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>15</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>16</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>17</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>18</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>19</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>20</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>21</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>22</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>23</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>24</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>25</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>26</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>27</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>28</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>29</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>30</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>31</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_4"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,18,1,1">
              <property name="spacing">
               <number>4</number>
              </property>
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>11</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Path</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_5"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>2</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>10</width>
                  <height>5</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>20</width>
                  <height>20</height>
                 </size>
                </property>
                <property name="text">
                 <string>...</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>14</pointsize>
                  <weight>75</weight>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="text">
                 <string>Download</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Maximum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>59</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="title">
             <string>Progress</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Maximum</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>57</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../image.qrc"/>
  <include location="../image.qrc"/>
 </resources>
 <connections/>
</ui>
